### PR TITLE
Gov partner details

### PR DESCRIPTION
**Pull request by marko911**
Details page for governments

### CODE REVIEW

File: pmp/app-shell.html [hunk 1]
```diff
@@ -194,9 +198,10 @@
           </template>
           <page-not-found name="not-found"></page-not-found>
 
-        </iron-pages>
-
-        <page-footer></page-footer>
+          </iron-pages>
+          <page-footer></page-footer>
+        </app-header-layout>
+      </app-drawer-layout>
```
Comment (robertavram) [inline pmp/app-shell.html]: delete last app-header-layout

File: pmp/pages/page-governments.html [hunk 2]
```diff
@@ -4,11 +4,13 @@
 <link rel="import" href="../../bower_components/paper-tabs/paper-tabs.html">
 <link rel="import" href="../../bower_components/paper-button/paper-button.html">
 
+<link rel="import" href="../app-elements/page-header/page-header.html">
```
Comment (robertavram) [inline pmp/pages/page-governments.html]: remove

File: pmp/pages/page-governments.html [hunk 3]
```diff
@@ -4,11 +4,13 @@
...
+<link rel="import" href="../app-elements/page-footer/page-footer.html">
+
 
 <!-- Replace these with something more apppriate -->
 <link rel="import" href="../app-elements/partners/partners-data/partners-list-data.html">
-<!-- <link rel="import" href="../app-elements/agreements/agreements-list.html"> -->
-<link rel="import" href="../app-elements/governments/governments-list.html">
+<link rel="import" href="../app-elements/agreements/agreements-list.html">
```
Comment (robertavram) [inline pmp/pages/page-governments.html]: add gov-list back.. not sure if you need agreements-list

File: pmp/pages/page-partners.html [hunk 4]
```diff
@@ -99,14 +99,11 @@ <h1 main-title>
           <partner-overview partner="[[partner]]"></partner-overview>
         </div>
         <div class="page" name="details">
-          <partner-details 
-              partner="[[partner]]" 
-              edit-mode="[[_hasEditPermissions(permissions)]]"
-              on-save-partner="_savePartner">      
-          </partner-details>
+          <government-details></government-details>
```
Comment (robertavram) [inline pmp/pages/page-partners.html]: remove this

File: scripts/lodash/lodash.html [hunk 5]
```diff
@@ -1,5 +1,3 @@
-<dom-module id="lodash">
-
-  <script src="../../bower_components/lodash/lodash.min.js"></script>
-
+<dom-module id="lodash4">
+    <script src="lodash4/lodash.min.js"></script>
```
Comment (robertavram) [inline scripts/lodash/lodash.html]: check where you're using the new methods.. see if you can fall back to 3.x

**Comment by adi130987** [comment]
@marko911 why was this closed?

**Comment by marko911** [comment]
@adi130987 accident

**Pull request merged**